### PR TITLE
Sequence compression

### PR DESCRIPTION
**Pull request by LBoleij**
Now using MapDB to store sequences

### CODE REVIEW

File: src/main/java/gui/MenuController.java [hunk 1]
```diff
@@ -51,7 +51,7 @@
     private GraphicsContext gc;
     private gui.GraphDrawer drawer;
     private SequenceGraph graph;
-    private HashMap<Integer, String> sequenceHashMap;
+    private HashMap<Integer, byte[]> sequenceHashMap;
```
Comment (BugOrFeature) [inline src/main/java/gui/MenuController.java]: This is unused.

File: src/main/java/parser/GfaParser.java [hunk 2]
```diff
@@ -3,58 +3,93 @@
...
-     * @param filepath A string specifying where the file is stored
+     * @param filePath A string specifying where the file is stored.
      * @return Returns a sequenceGraph
      * @throws IOException For instance when the file is not found
      */
-    public SequenceGraph parseGraph(String filepath) throws IOException {
-        sequenceHashMap = new HashMap<Integer, String>();
+    @SuppressWarnings("Since15")
+    public SequenceGraph parseGraph(String filePath) throws IOException {
+        String partPath = filePath.substring(filePath.length()-10, filePath.length());
```
Comment (BugOrFeature) [inline src/main/java/parser/GfaParser.java]: fix this hacky code

File: src/main/java/parser/GfaParser.java [hunk 3]
```diff
@@ -65,4 +100,55 @@ else if (line.startsWith("L")) {
         headers.add(header2);
         return headers;
     }
+
+    /**
+     * Try for encoding sequences (not used (yet)).
+     * @param seq The sequence to encode.
+     * @return The encoded sequence.
+     */
+    public String encodeSequence(String seq) {
```
Comment (BugOrFeature) [inline src/main/java/parser/GfaParser.java]: This is unused

File: src/main/java/gui/MenuController.java (hunk 1 above)
Comment (LBoleij) [inline src/main/java/gui/MenuController.java]: fixed

File: src/main/java/parser/GfaParser.java (hunk 2 above)
Comment (LBoleij) [inline src/main/java/parser/GfaParser.java]: Fixed

File: src/main/java/parser/GfaParser.java (hunk 3 above)
Comment (LBoleij) [inline src/main/java/parser/GfaParser.java]: Fixed

**Comment by BugOrFeature** [comment]
Please run maven before pushing

**Comment by BugOrFeature** [comment]
LGTM RTM

**Pull request merged**